examples/apps/collaborative-textarea/jest.config.js

```diff
@@ -20,6 +20,16 @@ module.exports = {
 	transform: {
 		"^.+\\.ts?$": "ts-jest",
 	},
+	reporters: [
+		"default",
+		[
+			"jest-junit",
+			{
+				outputDirectory: "nyc",
+				outputName: "jest-junit-report.xml",
+			},
+		],
+	],
```

Comment:
Shouldn't this have been caught in the other PR?

Reply:
Well, fluid-build isn't release yet, so not enforced by the CI. I must have missed it in the other CI.

Reply:
Ah yes, consuming the new version. Forgot 😄 .